### PR TITLE
twitter:added streaming and sentiment, iterate

### LINKED ISSUE #17: Get Tweets
- [ ] get last 7 days for each character (or more, if somehow possible)
- [ ] get new tweets with streaming API

### PR DESCRIPTION
**Pull request by ghost**
Fixes #17

### CODE REVIEW

**Comment by julienschmidt** [comment]
Run `npm test` locally the see what jshint (that's what Travis currently runs) complains about. Fix it and add another commit.

**Comment by julienschmidt** [comment]
```
     53 |            retext().use(sentiment).use(function() {

                                                 ^ Don't make functions within a loop.

     75 |            document.save(function(err, data) {

                                   ^ Don't make functions within a loop.
```

Still not fixed. Move the function definition outside the loop.

File: crawler/twitter.js [hunk 1]
```diff
@@ -27,59 +45,91 @@ exports.fetchTweets = function(query, maxID, i) {
...
-        //console.log(" printing id :"+data.statuses[i].id);
-
-        for (var count = 0; count < data.search_metadata.count; count++) {
-            console.log(data.statuses[count].lang,
-                data.statuses[count].created_at,
-                data.statuses[count].id,
-                data.statuses[count].text,
-                data.statuses[count].retweet_count,
-                data.statuses[count].favorite_count);
+            retext().use(sentiment).use(function() {
```
Comment (julienschmidt) [inline crawler/twitter.js]: You should move the analysis part in a new function. You repeat the same code in line 103.

Comment (ghost) [inline crawler/twitter.js]: working on it.

**Comment by ghost** [comment]
resolved!

**Pull request merged**